src/Billing/Services/Implementations/ProviderEventService.cs

```diff
@@ -79,7 +79,7 @@ public async Task TryRecordInvoiceLineItems(Event parsedEvent)
                         ClientName = client.OrganizationName,
                         PlanName = client.Plan,
                         AssignedSeats = client.Seats ?? 0,
-                        UsedSeats = client.UserCount,
+                        UsedSeats = (client.Seats ?? 0) - (client.AssignedSeats ?? 0),
```

Comment:
Won't this calculate the `RemainingSeats` rather than `UsedSeats`? I assume `UsedSeats` is synonymous with `AssignedSeats`.

Reply:
@eliykat This is just a weird naming issue because the contexts are different. In this context, `AssignedSeats` is how many seats the Provider has assigned to the Client Organization in question. The `UsedSeats` is how many seats the Client Organization has actually used by inviting users. 

In the other context of the SQL View, `UsedSeats` is the number of confirmed users and `AssignedSeats` (to be renamed as `OccupiedSeats` is the number of invited users.

Reply:
Good catch, thank you. Resolved here: https://github.com/bitwarden/server/pull/4446/commits/c9ebe060db48d1fb3e0e310c4c5d24066389084b